src/utils/Parallel.cpp

```diff
@@ -304,7 +304,7 @@ void Parallel::splitCommunicator(const std::string &groupName)
         accessorGroups[groupMap[name]].size++;
       }
     }
-    // Step 2 send groupCount to Master
+    // Step 2 send groupCount to Primary
```

Comment:
```suggestion
    // Step 2 send groupCount to primary rank
```
similar comments below